velox/expression/ConjunctExpr.cpp

```diff
@@ -128,7 +128,9 @@ void ConjunctExpr::evalSpecialForm(
   assert(activeRows); // lint
   int32_t numActive = activeRows->countSelected();
   for (int32_t i = 0; i < inputs_.size(); ++i) {
-    VectorPtr inputResult;
+    VectorPtr inputResult = nullptr;
+    auto releaseInputResult =
```

Comment:
nit: This is a little bit verbose. I wonder if we could introduce a helper class that could be used like this:

```
TempVectorPtr inputResult(context);

...inputResult.ptr() // Returns VectorPtr&
```